lib/diagnostic.rb

```diff
@@ -105,7 +126,12 @@ def eat_meat(food)
 end
```

Comment:
the say_hello method isn't defined on `HouseCat` 
so Ruby looks at `Cat` because `HouseCat` inherits from `Cat`
Ruby can't find the method on `Cat` but it looks on `Animal` because `Cat` inherits from `Animal`
Inheritance is so great!